verification/abi.go

```diff
@@ -3,11 +3,12 @@
 package verification
 
 import (
-	"errors"
 	"fmt"
 	"reflect"
 )
 
+var DefaultContextHandle = ContextHandle{0}
```

Comment:
nit: I think this can be const

Reply:
Checked this, seems golang doesnt allow arrays to be constants.